src/sqlfluff/core/templaters/placeholder.py

```diff
@@ -41,6 +41,10 @@
     "dollar": regex.compile(
         r"(?<![:\w\x5c])\${?(?P<param_name>[\w_]+)}?", regex.UNICODE
     ),
+    # e.g. WHERE bla = $name or WHERE bla = ${name} (DbUp compatible)
```

Comment:
I think this might be what you've intended here.
```suggestion
    # e.g. WHERE bla = $name$ (DbUp compatible)
```